docs/configuration.md

```diff
@@ -0,0 +1,85 @@
+# Ariadne configuration
+
+Ariadne uses the [Dhall](https://github.com/dhall-lang/dhall-lang)
+language for its configuration files and command line options to
+override values from files. Example configuration can be
+found in the [config/](config) directory.
+
+Path to the configuration file can be specified using `--config` option. By
+default `<XdgConfig>/ariadne/ariadne-config.dhall` is used, where
+`<XdgConfig>` is a system-dependent standard directory for local
+configuration files (`~/.config` on Linux or MacOS, `%APPDATA%` on
+Windows). If some value in the configuration is a relative file path,
+it's treated as relative to the directory in which main configuration
+file (e. g. `ariadne-config.dhall`) is located. There are special
+values which can be used in configuration as file paths:
+* `@DATA` is a system-dependent standard directory for local
+data (`~/.local/share` on Linux or MacOS, `%APPDATA%` on
+Windows) suffixed with `/ariadne`.
+* `@PWD` is the directory from which Ariadne executable is launched.
+
+If configuration file is not found, default configuration will be used
+which is compatible with Cardano Mainnet.
+
+Each option from the configuration can be overridden from the command
+like. Use `--help` flag for more details.
+
+Ariadne configuration consists of configurations of several
+components: Cardano, Wallet, Update, History. In the following we
+describe configuration of each component in detail.
+
+## Cardano configuration
+
+Cardano configuration contains parameters which are used by the
+underlying Cardano node.
+
+`configuration-options` defines `Configuration` used by Cardano
+node. For more details on that please read the `docs/configuration.md`
+file in the `cardano-sl` repository. If `configuration-path` doesn't
+exist, static configuration will be dumped to that path.
+
+`log-config` is the path to the logging configuration (as used by
+`log-warper`). If it's not provided a reasonable default configuration
+will be used.
+
+`log-prefix` is the file prefixed to all paths in logging
```

Comment:
Maybe s/file/string?

Reply:
I'd say s/file/path. Because it's prefixed as a path, with separator.

Reply:
Also probably s/prefixed/prepended.